packages/nouns-contracts/contracts/libs/Noracle.sol

```diff
@@ -95,13 +80,7 @@ library Noracle {
     ) internal {
         uint32 currentIndex = self.index;
         uint32 cardinality = self.cardinality;
-        uint32 cardinalityNext = self.cardinalityNext;
-        if (cardinalityNext == 0) return;
-
-        // if the conditions are right, we can bump the cardinality
-        if (cardinalityNext > cardinality && currentIndex == (cardinality - 1)) {
```

Comment:
@eladmallel are you sure `cardinalityNext` is not needed?
My understanding is that it's needed so that `observe` works correctly.

what happens in this case:
`self.cardinality = 10`
and `self.index = 0`
assuming all the 10 slots are filled with data

then we call `grow(20)`
now slows 10-19 are empty
but if we call `observe` it will look at slot 0, then slot 19 (going backwards), see an empty slot, and quit.
that's not what we'd want, right?